src/services/identity/NotificationsService.ts

```diff
@@ -218,31 +196,30 @@ class NotificationsService {
           model: Site,
           as: "site",
           required: true,
-          include: [
-            {
-              model: Repo,
-              required: true,
-              where: {
-                name: siteName,
-              },
-            },
-          ],
+          where: {
+            id: siteMember.siteId,
+          },
         },
       ],
     })
 
     if (recentTargetNotification) {
       // Update existing notification
-      await recentTargetNotification.update({
-        firstReadTime: null,
-        createdAt: new Date(),
-      })
+      // createdAt is a special column which must be flagged as changed
+      recentTargetNotification.changed("createdAt", true)
+      await recentTargetNotification.update(
+        {
+          firstReadTime: null,
+          createdAt: new Date(),
+        },
+        { raw: true }
```

Comment:
why do we require this flag?

Reply:
`createdAt` seems to be a special param because it's automatically generated - it can't seem to be edited normally, so we use raw to allow the normal setter to be ignored

See https://sequelize.org/v3/api/instance/#setkey-value-options!